### PR TITLE
Update github files

### PR DESCRIPTION
**Pull request by isuruf**
Fixes https://github.com/conda-forge/conda-smithy/issues/704

### CODE REVIEW

File: conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,46 +1,25 @@
...
-<details>
-
-
-
-</details>
-
-<br>
-Environment (`conda env export`):
-<br>
-<br>
```
Comment (jakirkham) [inline conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: This was mainly here as there are some annoying bugs in GitHub when rendering several `details` sections. So if we only want one, that should be fine. Just figured I'd mention it to provide context.

File: conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,46 +1,25 @@
+<!--
 Thanks for reporting your issue.
-
-
 Please fill out the sections below.
+-->
+Issue
```
Comment (jakirkham) [inline conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: nit: Maybe add a `:` at end.

File: conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (isuruf) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: I checked both before sending the PR. Maybe github fixed their bugs

**Comment by jakirkham** [comment]
Thanks @isuruf. Agree this looks better. Only one nit came to mind. Otherwise it seems fine.

File: conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,46 +1,25 @@
...
-
-</details>
-
-<br>
-Environment (`conda env export`):
-<br>
-<br>
-
+<br/>
+Environment (<code>conda list</code>):
```
Comment (jakirkham) [inline conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: So the reason for `conda env export` is this creates a nice format that we can trivially recreate a `conda` environment from locally. AFAIK there is no way to do this with `conda list`. Do you know of such a way?

Comment (isuruf) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: Nope. Main reason for the change was that `conda env export` doesn't give you the channel that a package came from and that is important information.

Comment (jakirkham) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: @kalefranz, do you have any suggestions for us?

Comment (msarahan) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: conda list --explicit > somefile.txt

**Comment by ocefpaf** [comment]
LGTM. Let's address the comments here in another PR.

Thanks @isuruf! This is awesome BTW!!

File: conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (kalefranz) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: 1. `conda list --explicit`
2. `conda list --export`
3. `conda env export`

Right now, no One Right Way.  That's just where we are at the moment.

Comment (jakirkham) [reply conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md]: 1 solves the issue of reproducibly creating environments, but is very unreadable (as it contains full URLs).

2 doesn't seem to have the channel information AFAICT.

3 is ideal IMHO, but hasn't listed the channels for individual packages for some time. ( https://github.com/conda/conda/issues/5706 )

**Pull request merged**